packages/office-ui-fabric-react/src/components/Slider/Slider.types.ts

```diff
@@ -123,6 +123,24 @@ export interface ISliderProps extends React.ClassAttributes<SliderBase> {
    * @defaultvalue false
    */
   originFromZero?: boolean;
+
+  /**
+
+   * Optional flag to render the thumblabel.
+
+   */
+
+  thumblabel?: boolean;
+
+  /**
+
+   * Optional flag to indicate whether to have tick marks
+
+   * @defaultvalue false
+
+   */
+
+  marks?: boolean;
```

Comment:
I would recommend adding also a render override so that users can provide their own ticks/marks. There is plenty of examples on how you can achieve that. The type should be of `IRenderFunction` that will also pass back the props and the defaultRender.

Reply:
here maybe `enableMarks` would be a better hinting name. A `defaultvalue` doc filed would be nice to have to.

Reply:
I looked at the example of IRenderFunction but it's unclear how to use it in this case. Can you clarify what you mean in this context?

Reply:
An optional override will give options to the user to override the default regular tick but I suppose that an be added later when the need for it comes in the shape of a feature request. It’s not a blocking feedback.